salt/states/grains.py

```diff
@@ -144,17 +154,6 @@ def list_present(name, value, delimiter=DEFAULT_TARGET_DELIM):
             if set(value).issubset(set(__salt__['grains.get'](name))):
                 ret['comment'] = 'Value {1} is already in grain {0}'.format(name, value)
                 return ret
-            elif name in __context__.get('pending_grains', {}):
```

Comment:
If you wouldn't mind submitting a new PR, that would be great! If not, we can grab it in the next couple of days.